src/integration-test/java/uk/gov/ida/apprule/EidasUserLogsInIntegrationTests.java

```diff
@@ -63,7 +63,7 @@ public void debugPageLoadsAndValuesForOptionalAttribuesAreReturnedTest() {
         if (requestGender) { assertThat(page).contains(IdaConstants.Eidas_Attributes.Gender.NAME); }
         final String samlResponse = authnRequestSteps.eidasUserConsentsReturnSamlResponse(cookies, false);
         final InboundResponseFromCountry inboundResponseFromCountry = samlDecrypter.decryptEidasSaml(samlResponse);
-        assertThat(inboundResponseFromCountry.getIssuer()).isEqualTo("http://localhost:0/stub-country/ServiceMetadata");
+        assertThat(inboundResponseFromCountry.getIssuer()).isEqualTo("http://localhost:0/"+EIDAS_SCHEME_NAME+"/ServiceMetadata");
```

Comment:
It always bothers me when whitespaces are lacking... We'll add it back the next time we make changes to the stub IDP

Reply:
👍